frontend/src/components/Navbar.tsx

```diff
@@ -89,18 +97,22 @@ function Navbar(props: any) {
   // eslint-disable-next-line consistent-return
   const handleKeyDown = (event: any) => {
     if (event.key === 'Enter') {
-      submitSearch(currFilters);
-      return history.push('/Browse');
+      if (!currFilters.query.length) {
+        alert('Please input keyword into search bar.');
```

Comment:
I think  personally I would recommend we avoid an alert, just prevent the search

Reply:
Ok I'll remove that! Also yeah the repeating image is a mistake, the next one is gonna be slightly better (in a mo)